.travis.yml

```diff
@@ -51,7 +51,7 @@ deploy:
   api_key:
     secure: 2flghVwIh/AVn5hXljg4qjRdQfL7u/0PUiFvFumz8Q3UPkzsT3rQ+0GjSV6D9XLnnVNFK0KZ2eZEji1Zh2wht3V14FkHq95kRin1V82Xl69b0DD0S9fq9py/SLX3cR1Fr4FsCvCQL2VLkTmHoLZ+chiWHjR/Em0oro8/4tCfShk1mgQ8ZE7o3wnwItXRckPdtTj8Gzug3ce/lFmiHgKz9I8PFu9ErqkKmKQsitzuZwe58xRz3WOWsYrw3qNYjuC5TUnXKcHSwmI2F6WoAgO5latia2xryx9yosxCaZL7qie0kgW/lhk62FY1RwYzXuKfb4pW+XQBnJhpr7uBhq2K3HVWdYuUqTf/i6rYTIm2x9LuxkY25ok0V3vpKQDOwALEVSZpMMvNPRwyLWuUecNtNz73V47BNwETSo45dBCNgjhvu2Zh19WTHpHEgYacztlcUR/F9heRa1RDEKalS+XbtLX7NgyejJXARG9zb++G1d37wc3tESPge8Nwj5/ls3jN5M5pFE1/K7Ayus1MLBDsTBKLshmbfWKq7WbiXWiQtsou0PT0NJKojk+j7KCTYMg5nt79RF0OaRWsdog7FjE/KmnxDCjgQp8P1VEV0olvtDPbr52WUUVUofpNnV8V/70AI2eBkm4ZK0/8C2wBOKMH7XwYTVVyboaqwIpY+RDEi/Y=
   on:
-    all_branches: true
+    branch: 'master'
```

Comment:
This is not a good idea for a number of reasons. First off, it prevents you from being able to publish a bugfix to a previous major, using a maintenance branch. Secondly, I believe it will prevent the package from ever publishing. 

In order to support `none` in `pr-bumper` some changes were made to the way the builds work. Now, when a PR is merged, three builds kick off:
1) The merge commit from the PR is built on the branch the PR is made into, this build does the `pr-bumper bump` step and the `Automated version bump` commit is made, which creates a tag with the new version
2) The newly created commit is built from the target branch (usually `master` but maybe something like `0.x` if there's a maintenance branch, this build is pretty much ignored, all the `maybe-do-this.sh` scripts look for the `Automated version bump` commit message and stop executing early if they see it.
3) The actual new tag is built, this is one also skips a lot of things, like the tests, etc b/c of the `maybe-do-something.sh` scripts, but then satisfies the `on` clauses for `deploy` and finally publishes the package to `npm`. 

With `branch: 'master'` and `tags: true` you're essentially making it so that the `on` is never satisfied.